content/en/docs/pipelines/installation/overview.md

```diff
@@ -131,21 +151,27 @@ Interfaces
   * Kubeflow Pipelines UI via the **Open Pipelines Dashboard** link in the 
     GCP Console
   * Kubeflow Pipelines SDK in Cloud Notebooks
+  * Kubeflow Pipelines endpoint of your instance is auto-configured for you.
 
 Release Schedule
 : AI Platform Pipelines is available for a chosen set of stable Kubeflow
 Pipelines releases. You will receive updates slightly slower than Kubeflow
 Pipelines Standalone.
 
 Upgrade Support (**Alpha**)
-: In place upgrade is not supported. Reinstall new version to upgrade is supported without official documentation. You can [delete a
-AI Platform Pipelines application](https://cloud.google.com/ai-platform/pipelines/docs/getting-started#clean_up) **WITHOUT** selecting **Delete cluster** checkbox and
-reinstall the newer version in the same cluster and same namespace. Persisted
-artifact and database data are in GCP persistent disks and will be automatically
-picked up during reinstallation.
+: In place upgrade is not supported. Reinstall new version while retaining data
+is supported without official documentation. You can
+[delete a AI Platform Pipelines application](https://cloud.google.com/ai-platform/pipelines/docs/getting-started#clean_up)
+**WITHOUT** selecting **Delete cluster** checkbox and reinstall the newer version
+in the same cluster and the same namespace. Persisted artifact and database data
+are in GCP persistent disks and will be automatically picked up during
+reinstallation.
 
 Notes on specific features
-: After deployment, your Kubernetes cluster contains Kubeflow Pipelines only. 
+: 
+  * After deployment, your Kubernetes cluster contains Kubeflow Pipelines only. 
   It does not include the other Kubeflow components. 
   For example, to use a Jupyter Notebook, you can use [AI Platform 
   Notebooks](https://cloud.google.com/ai-platform/notebooks/docs/).
+  * KFP multi user support is **not available** in AI Platform Pipelines, because
+  it depends on other Kubeflow components.
```

Comment:
I thought you said that multi-user support will be available in v1.1. This looks inconsistent.

Reply:
It's available in (full) Kubeflow 1.1, but not the other two options, because the other two options don't deploy full Kubeflow.